webapp/app.py

```diff
@@ -232,24 +228,11 @@
 )
 app.add_url_rule("/pro/subscribe", view_func=advantage_shop_view)
 app.add_url_rule("/pro/subscribe/blender", view_func=blender_shop_view)
-app.add_url_rule("/pro/subscribe/thank-you", view_func=advantage_thanks_view)
-app.add_url_rule(
-    "/pro/subscribe",
-    view_func=post_advantage_subscriptions,
-    methods=["POST"],
-    defaults={"preview": False},
-)
 app.add_url_rule(
     "/pro/subscribe",
     view_func=cancel_advantage_subscriptions,
     methods=["DELETE"],
 )
-app.add_url_rule(
-    "/pro/subscribe/preview",
-    view_func=post_advantage_subscriptions,
-    methods=["POST"],
-    defaults={"preview": True},
-)
```

Comment:
I think since this code here is removed I get this error when resizing subscription on /pro/dashboard.

![image](https://github.com/canonical/ubuntu.com/assets/57550290/01484db7-6bc5-40bb-94f8-90d49b28d218)